.cspell.json

```diff
@@ -101,7 +109,25 @@
         "Useds",
         "venv",
         "VNET",
-        "XLSX"
+        "XLSX",
+        "octocat",
+        "ministryofjustice",
+        "outro",
+        "GUIDLINES",
+        "nofollow",
+        "noarchive",
+        "noimageindex",
+        "exceljs",
+        "nbsp",
+        "srcs",
+        "inmemory",
+        "AREED",
+        "EHRC",
+        "EASS",
+        "UNSUPPORTRED",
+        "acode",
```

Comment:
I can see the typo here, there's a test description that should be `... a-code ...`, save adding this line.

Reply:
Have updated.